tools/lkl/lib/hijack/init.c

```diff
@@ -177,6 +180,26 @@ void fixup_netdev_linux_fdnet_ops(void)
 	lkl_netdev_linux_fdnet_ops.eventfd = dlsym(RTLD_NEXT, "eventfd");
 }
 
+static void PinToCpus(const cpu_set_t* cpus) {
+	if (sched_setaffinity(0, sizeof(cpu_set_t), cpus)) {
+		perror("sched_setaffinity");
+	}
+}
+
+static void PinToFirstCpu(const cpu_set_t* cpus) {
```

Comment:
Minor thing regarding coding style, it would be great if you could you fix up this and the function above.